File: rtcbuilder/tags/RELEASE_1_0_0_RC1/jp.go.aist.rtm.rtcbuilder.csharp/resource/AIST4/MyServiceImpl.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Test
{
    public class MyServiceImpl : MarshalByRefObject, MyService
    {
        public System.String echo(System.String msg)
        {
            throw new Exception("The method or operation is not implemented.");
        }
        public System.String[] get_echo_history()
        {
            throw new Exception("The method or operation is not implemented.");
        }
        public void set_value(System.Single value)
        {
            throw new Exception("The method or operation is not implemented.");
        }
        public System.Single get_value()
        {
            throw new Exception("The method or operation is not implemented.");
        }
        public System.Single[] get_value_history()
        {
            throw new Exception("The method or operation is not implemented.");
        }
    }
}

File: rtcbuilder/jp.go.aist.rtm.rtcbuilder.vbdotnet/resource/AIST1/MyServiceImpl.vb
Public Class MyServiceImpl
    Inherits MarshalByRefObject
    Implements MyService
    Public Function echo(ByVal msg As System.String) As System.String Implements MyService.echo
        Throw New Exception("The method or operation is not implemented.")
    End Function

    Public Function get_echo_history() As System.String() Implements MyService.get_echo_history
        Throw New Exception("The method or operation is not implemented.")
    End Function

    Public Sub set_value(ByVal value As System.Single) Implements MyService.set_value
        Throw New Exception("The method or operation is not implemented.")
    End Sub

    Public Function get_value() As System.Single Implements MyService.get_value
        Throw New Exception("The method or operation is not implemented.")
    End Function

    Public Function get_value_history() As System.Single() Implements MyService.get_value_history
        Throw New Exception("The method or operation is not implemented.")
    End Function

End Class
